src/Glottis20Mint.sol

```diff
@@ -94,26 +94,34 @@ contract Glottis20Mint is ReentrancyGuard {
         uint256 liquidityEth = ethLiquidity.rawSub(protocolEthFee).rawSub(creatorEthFee).rawSub(callerEthFee);
 
         // Transfer fees
+        // fico23:
+        // why are we forcing? why not just SafeTransferLib.safeTransferETH?
+        // force will call selfdestruct - so im not sure how it works with 3 in a row
+        // even if it does work, its better to use safe transfer
         protocolWallet.forceSafeTransferETH(protocolEthFee);
         readTokenCreator(token).forceSafeTransferETH(creatorEthFee);
         msg.sender.forceSafeTransferETH(callerEthFee);
 
         glottis20.mint(address(this), currentSupply);
 
+        // fico23:
+        // why do we approve 0 then max? shouldn't approve.max be enough?
```

Comment:
from open zeppelin docs om approval:

<head></head>
  | Beware that changing an allowance with this method brings the risk that someone may use both the old and the new allowance by unfortunate transaction ordering. One possible solution to mitigate this race condition is to first reduce the spender’s allowance to 0 and set the desired value afterwards: https://github.com/ethereum/EIPs/issues/20#issuecomment-263524729
-- | --




technically we are not using open zeppelin but just to be sure? maybe it does not matter since we are in the same block but decided to follow advice nevertheless.